lib/src/core/channel/thread_channel.dart

```diff
@@ -77,6 +77,23 @@ class ThreadMember extends SnowflakeEntity implements IThreadMember {
   }
 }
 
+abstract class IThreadMemberWithMember extends IThreadMember {
+  /// Fetched member from API
+  Member get fetchedMember;
+}
+
+class ThreadMemberWithMember extends ThreadMember implements IThreadMember {
+  late final Member fetchedMember;
+
+  ThreadMemberWithMember(INyxx client, RawApiMap raw, Cacheable<Snowflake, IGuild> guild) : super(client, raw, guild) {
+    fetchedMember = Member(client, raw['client'] as RawApiMap, guild.id);
```

Comment:
```suggestion
    fetchedMember = Member(client, raw['member'] as RawApiMap, guild.id);
```

The field name here is incorrect:
> When `with_member` is set to `true`, the thread member object will include a `member` field containing a [guild member](https://discord.com/developers/docs/resources/guild#guild-member-object) object.